storm-core/src/jvm/org/apache/storm/metric/internal/CountStatAndMetric.java

```diff
@@ -197,12 +196,6 @@ long readApproximateTime(long value, long timeSpent, long[] bucketTime, long[] b
         return total;
     }
 
-    public void close() {
-        if (_task != null) {
-            _task.cancel();
```

Comment:
@revans2 - As far as I could see, close is being called only in the render-stats. Tests initialize with negative number of buckets so _task is never initialized at all.